tests/console/commands/env/test_info.py

```diff
@@ -25,27 +25,21 @@ def tester(command_tester_factory):
 def test_env_info_displays_complete_info(tester):
     tester.execute()
 
-    expected = """
+    expected = f"""
 Virtualenv
 Python:         3.7.0
 Implementation: CPython
-Path:           {prefix}
-Executable:     {executable}
+Path:           {Path('/prefix')}
+Executable:     {sys.executable}
 Valid:          True
 
 System
 Platform:   darwin
 OS:         posix
-Python:     {base_version}
-Path:       {base_prefix}
-Executable: {base_executable}
-""".format(
-        prefix=str(Path("/prefix")),
-        base_prefix=str(Path("/base/prefix")),
-        base_version=".".join(str(v) for v in sys.version_info[:3]),
-        executable=sys.executable,
-        base_executable="python",
-    )
+Python:     {'.'.join(str(v) for v in sys.version_info[:3])}
```

Comment:
Any thoughts on this would be preferable?

```suggestion
Python:     {sys.version.split(" ")[0]}
```

Reply:
personally, it's less readable and less explicit to me. as is, ik at-a-glance exactly what's happening. whereas i'd have to execute the latter to understand